src/site/content/en/fast/reduce-webfont-size/index.md

```diff
@@ -53,145 +39,88 @@ Today there are four font container formats in use on the web:
 * [WOFF](https://en.wikipedia.org/wiki/Web_Open_Font_Format)
 * [WOFF2](https://www.w3.org/TR/WOFF2/).
 
-[WOFF](http://caniuse.com/#feat=woff) and [WOFF 2.0](http://caniuse.com/#feat=woff2) enjoy the widest support,
-however for compatibility with older browsers you may need to include other formats:
+[WOFF](http://caniuse.com/#feat=woff) and [WOFF 2.0](http://caniuse.com/#feat=woff2) enjoy the widest support, and WOFF 2.0 is supported by all modern browsers (excluding Internet Explorer 11).
 
-* Serve WOFF 2.0 variant to browsers that support it.
-* Serve WOFF variant to the majority of browsers.
-* Serve TTF variant to old Android (below 4.4) browsers.
-* Serve EOT variant to old IE (below IE9) browsers.
+* Serve WOFF 2.0 variant to modern browsers.
+* If absolutely necessary&mdash;such as if you still need to support Internet Explorer 11, for example&mdash;serve the WOFF as a fallback.
```

Comment:
```suggestion
* If absolutely necessary&mdash;such as if you still need to support Internet Explorer 11, for example&mdash;serve the WOFF as a fallback.
* Alternatively, consider not using web fonts for legacy browsers and falling back to the system fonts. This may be more performant for older, more constrained, devices as well.
```